students.txt

```diff
@@ -7,7 +7,9 @@ List of students
 
 1. Anastasia P.
 2. Natalia T.
-3. Jorge T.
+3. George T.
```

Comment:
You forgot to expand you last name. Please add full Last name instead of initial letter.